services/src/main/java/edu/unc/lib/dl/cdr/services/processing/SetAsPrimaryObjectService.java

```diff
@@ -70,8 +70,7 @@ public void setAsPrimaryObject(AgentPrincipals agent, PID fileObjPid) {
         work.setPrimaryObject(fileObjPid);
 
         // Send message that the action completed
-        operationsMessageSender.sendIndexingOperation(agent.getUsername(), Arrays.asList(fileObjPid),
-                IndexingActionType.SET_AS_PRIMARY_OBJECT);
+        operationsMessageSender.sendSetAsPrimaryObjectOperation(agent.getUsername(), Arrays.asList(fileObjPid));
```

Comment:
This message should include the parent and the file object (the parent is actually what needs to be reindexed).

Reply:
do you want both pids included, or shall i simply pass the parent pid and then have the method retrieve the child pid to add to the message before it's sent?

Reply:
I would provide both pids to the method

Reply:
done